admin/script/supplier_input.js

```diff
@@ -0,0 +1,16 @@
+$('#formSupplier').submit(function(e){
+    e.preventDefault();
+    var formData = new FormData(this);
+    $.ajax({
+        type: 'POST',
+        url: host+"input_supplier.php",
+        data: formData,
+        cache: false,
+        contentType: false, 
+        processData: false, 
+        dataType: 'json',
+        success: (result) => {
+            console.log(result, url);
```

Comment:
@GebyRezeki , ini setelah sukses dan nerima respon dari BE ngapain lagi?, harusnya menampilkan pesan sukses, tambahkan ya